src/Symfony/Component/Workflow/Tests/SupportStrategy/ClassInstanceSupportStrategyTest.php

```diff
@@ -6,20 +6,26 @@
 use Symfony\Component\Workflow\SupportStrategy\ClassInstanceSupportStrategy;
 use Symfony\Component\Workflow\Workflow;
 
+/**
+ * @group legacy
```

Comment:
this file basically only needs this change.. what about `SubjectA` + `SubjectB` for the new test?

Reply:
done

Reply:
still.. no real reason to optimize/tweak a legacy test (`::class` notation, class removals). Doesnt hurt; just extra diff :)

Reply:
indeed, adding the `::class` notation was ok because the line was actually changed. Now it just gives merge conflicts, should be reverted